tests/app/views/contexts/summary/test_question.py

```diff
@@ -164,84 +285,45 @@ def test_create_question_with_answer_label_when_empty_title(self):
 
     def test_concatenate_textfield_answers(self):
         answer_separators = {"Newline": "<br>", "Space": " "}
+        self.schema = self.address_questionnaire_schema()
 
         for concatenation_type, concatenation_character in answer_separators.items():
             with self.subTest(
                 concatenation_type=concatenation_type,
                 concatenation_character=concatenation_character,
             ):
                 # Given
-                self.answer_store.add_or_update(
-                    Answer(answer_id="address-line-1", value="Cardiff Rd")
-                )
-                self.answer_store.add_or_update(
-                    Answer(answer_id="town-city", value="Newport")
-                )
-                self.answer_store.add_or_update(
-                    Answer(answer_id="postcode", value="NP10 8XG")
+                self.update_answer_store(False)
+
+                # When
+                question = self.address_question(concatenation_type)
+
+                # Then
+                self.assertEqual(
+                    question.answers[0]["value"],
+                    f"Main Building{concatenation_character}Cardiff Rd{concatenation_character}Newport{concatenation_character}NP10 8XG",
                 )
+                self.assertEqual(len(question.answers), 1)
 
-                address_line_1 = {
-                    "id": "address-line-1",
-                    "label": "Address line 1",
-                    "mandatory": False,
-                    "type": "TextField",
-                }
-                address_line_2 = {
-                    "id": "address-line-2",
-                    "label": "Address line 2",
-                    "mandatory": False,
-                    "type": "TextField",
-                }
-                town_city = {
-                    "id": "town-city",
-                    "label": "Town or City",
-                    "mandatory": False,
-                    "type": "TextField",
-                }
-                county = {
-                    "id": "county",
-                    "label": "County",
-                    "mandatory": False,
-                    "type": "TextField",
-                }
-                postcode = {
-                    "id": "postcode",
-                    "label": "Postcode",
-                    "mandatory": False,
-                    "type": "TextField",
-                }
+    def test_concatenate_textfield_answers_default(self):
```

Comment:
The other issue with testing concat is that, non concat isn't tested. Personally, I would make `_get_answer` non-private and test that, otherwise you will need another test for non concat with default.

Reply:
`_get_answer` non private and tested in non concat now. Cheers!